src/main/java/org/openforis/ceo/local/JsonGeoDash.java

```diff
@@ -110,11 +111,11 @@ public synchronized String createDashBoardWidgetById(Request req, Response res)
 
```

Comment:
can you return just "()" without and issue?  If so you can just set callback to "" instead of null
Also use ternary `return callback == null ? "" : callback + "()"`

Reply:
seems to also repeat 5 times

Reply:
No, if there is a callback sent it needs to call the actual callback function

Reply:
I removed all checks for the callback parameter since we no longer send it.